paywall/src/unlock.js/Wallet.ts

```diff
@@ -0,0 +1,315 @@
+import { Web3Window, web3MethodCall } from '../windowTypes'
+import { PostMessages } from '../messageTypes'
+import IframeHandler from './IframeHandler'
+import { PaywallConfig } from '../unlockTypes'
+import StartupConstants from './startupTypes'
+
+declare const process: {
```

Comment:
Since you do that in a lot of places can we just add this to  a types file? either `unlockTypes` or `startupTypes`?

Reply:
same story: I fixed it earlier, and forgot to push my commit.